app/views/landing_page/landing_page.erb

```diff
@@ -33,6 +33,12 @@
 
   <title><%= page["title"]["value"] %></title>
 
+  <script>
+    window.I18n = {};
+    <%= javascripts[:translations] %>
+  </script>
```

Comment:
I would move this `script` tag to line 141 right before including `application.js`. It doesn't need to be in head and it will slow down the rendering (not much) because the script tag content needs to be read and evaluated.